### PR TITLE
VP-2472: Preview product to match

### PR DESCRIPTION
**Pull request by AliveMen**
Add preview blade
Fix output tuning blade issues

### CODE REVIEW

File: src/VirtoCommerce.CatalogModule.Web/Scripts/blades/dynamicAssociations/dynamicAssociation-viewer.tpl.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+<script type="text/ng-template" id="export-item-list.row.html">
+    <div ng-click='grid.appScope.selectItem($event, row.entity)' ng-repeat="(colRenderIndex, col) in colContainer.renderedColumns track by col.uid"
+         ui-grid-one-bind-id-grid="rowRenderIndex + '-' + col.uid + '-cell'" class="ui-grid-cell"
+         ng-class="{'ui-grid-row-header-cell': col.isRowHeader, '__selected': row.entity.id===grid.appScope.selectedNodeId }"
+         role="{{col.isRowHeader ? 'rowheader' : 'gridcell' }}" ui-grid-cell>
+    </div>
+</script>
+
+<script type="text/ng-template" id="export-item-list-icon.cell.html">
```
Comment (yecli) [inline src/VirtoCommerce.CatalogModule.Web/Scripts/blades/dynamicAssociations/dynamicAssociation-viewer.tpl.html:46]: Do we need these?
Also please create independent templates, not override export ones, e.g. export-item-list.row.html

File: src/VirtoCommerce.CatalogModule.Web/Scripts/blades/dynamicAssociations/dynamicAssociations-detail.js [hunk 2]
```diff
@@ -72,17 +72,22 @@ angular.module('virtoCommerce.catalogModule')
         };
 
         $scope.outputTuning = function () {
+            const rulesBlock = _.find(blade.currentEntity.expressionTree.children, x => x.id === $scope.BlockMatchingRules);
```
Comment (yecli) [inline src/VirtoCommerce.CatalogModule.Web/Scripts/blades/dynamicAssociations/dynamicAssociations-detail.js:75]: I think categories for sorting should come from Resulting rule

File: src/VirtoCommerce.CatalogModule.Web/Scripts/blades/dynamicAssociations/dynamicAssociation-viewer.tpl.html (hunk 1 above)
Comment (AliveMen) [reply src/VirtoCommerce.CatalogModule.Web/Scripts/blades/dynamicAssociations/dynamicAssociation-viewer.tpl.html]: done

File: src/VirtoCommerce.CatalogModule.Web/Scripts/blades/dynamicAssociations/dynamicAssociations-detail.js (hunk 2 above)
Comment (AliveMen) [reply src/VirtoCommerce.CatalogModule.Web/Scripts/blades/dynamicAssociations/dynamicAssociations-detail.js]: changed

**Pull request merged**